Source/WebCore/page/Navigation.cpp

```diff
@@ -809,7 +867,10 @@ bool Navigation::innerDispatchNavigateEvent(NavigationNavigationType navigationT
         if (navigationType == NavigationNavigationType::Traverse)
             m_suppressNormalScrollRestorationDuringOngoingNavigation = true;
 
-        if (navigationType == NavigationNavigationType::Push || navigationType == NavigationNavigationType::Replace) {
+        if (navigationType == NavigationNavigationType::Reload) {
+            // Not in specification but matches chromium implementation and tests.
```

Comment:
Should we raise a spec issue?

Reply:
I filed https://github.com/whatwg/html/issues/10621.